src/components/Button/Button.stories.tsx

```diff
@@ -47,10 +47,17 @@ type Args = React.ComponentProps<typeof Button>;
 
 const Template: Story<Args> = (args) => <Button {...args} />;
 
+export const Default = Template.bind({});
```

Comment:
I moved the "secondary" story to the top and renamed it "default" so it makes a little more sense that when you drop a button in somewhere, it defaults to this style.